File: java/abstractfactory/ej1/NonLuxurySUV.java
package patrones.abstractfactory.ej1;

public class NonLuxurySUV implements SUV {
	private String name;

	public NonLuxurySUV(String sName) {
		name = sName;
	}

	public String getSUVName() {
		return name;
	}

	public String getSUVFeatures() {
		return "Non-Luxury SUV Features ";
	}
}

File: c#/abstractfactory/Ejm1/NonLuxurySUV.cs

namespace AbstractFactory.Ejm1
{
    public class NonLuxurySUV : SUV
    {
        private string Name;

        public NonLuxurySUV(string sName)
        {
            Name = sName;
        }

        public string GetSUVName()
        {
            return Name;
        }

        public string GetSUVFeatures()
        {
            return "Non-Luxury SUV Features ";
        }
    }
}
